src/iterators.jl

```diff
@@ -156,6 +159,47 @@ end
     reinit!(fv, fc.cc, fc.current_faceid[])
 end
 
+"""
+    InterfaceCache(grid::Grid)
+    InterfaceCache(dh::AbstractDofHandler)
+Create a cache object with pre-allocated memory for the coordinates and faces of an
+interface. The cache is updated for a new cell by calling `reinit!(cache, face_a, face_b)` where
+`face_a::FaceIndex` and `face_b::FaceIndex` are the interface faces.
+**Struct fields of `InterfaceCache`**
+ - `ic.face_a :: FaceCache`: current cell node coordinates
+ - `ic.face_b :: FaceCache`: neighbor cell node coordinates
+**Methods with `InterfaceCache`**
+ - `reinit!(cache::InterfaceCache, face_a::FaceIndex, face_b::FaceIndex)`: reinitialize [`InterfaceCache`](@ref)
+ - `cellid(ic)`: get cell ids of the current interface
+ - `getnodes(ic, use_cell_a = true)`: get the global node ids of cell A or cell B of the interface
```

Comment:
Perhaps the way to use this would be through `getnodes(ic.a)` and `getnodes(ic.b)` instead?